apps/system/js/nfc_handover_manager.js

```diff
@@ -325,23 +278,16 @@
      * @memberof NfcHandoverManager.prototype
      */
     _findPairedDevice: function _findPairedDevice(mac, foundCb, notFoundCb) {
-      this._debug('_findPairedDevice');
-      if (this.defaultAdapter == null) {
-        // No BT
-        this._debug('No defaultAdapter');
-        return;
-      }
-
-      var req = this.defaultAdapter.getPairedDevices();
-      req.onsuccess = () => {
-        var devices = req.result;
-        this._debug('# devices: ' + devices.length);
-        for (var i = 0; i < devices.length; i++) {
+      this.debug('_findPairedDevice');
+      Service.request('Bluetooth:getPairedDevices').then((devices) => {
+        this.debug('# devices: ' + devices.length);
+        for (var i = 0, len = devices.length; i < len; i++) {
           var device = devices[i];
-          this._debug('Address: ' + device.address);
-          this._debug('Connected: ' + device.connected);
+          this.debug('Address: ' + device.address);
+          // deprecated in BTv2
+          this.debug('Connected: ' + device.connected);
```

Comment:
nit: Suggest to remove the console log since it should be a JS error in Bluetooth API v2(`device.connected` is undefined).